css/pages/product.css

```diff
@@ -0,0 +1,28 @@
+.filter-container {
+  background: var(--color-white);
+
+  box-shadow: 0 0.1rem 0.4rem var(--color-slate-200);
```

Comment:
You can use px instead of rems for units less than 10px. This isn't mandatory, just a suggestion.

Reply:
Thanks for the suggestion,